src/views/Dashboard/Lab/Services/index.vue

```diff
@@ -216,6 +230,14 @@ export default {
     confirmDeleteService(item) {
        this.idItemDeleted = item.id
        this.showDialog = true
+    },
+
+    async checkServiceCount() {
+       const res = await queryServicesCountByOwnerId(this.api, this.pair.address)
+       this.serviceCount = Number(res.toHuman())
+       if (this.serviceCount <= 1) {
+          this.activeButton = false
+       }
```

Comment:
Wait, can't you get this data from the Vuex store?

Reply:
Try retrieving the lab services number from here: https://github.com/luthfiazhari/debio-frontend/blob/fdcfce85f5bdecdeeb928cb5a4f479b7f3d31af4/src/store/substrate/index.js#L427

Reply:
this issue has been worked on, got data from the store @agustinustheo